app/assets/javascripts/components/article_finder/article_finder_row.jsx

```diff
@@ -64,6 +67,18 @@ const ArticleFinderRow = createReactClass({
     return this.props.deleteAssignment(assignment);
   },
 
+  async isCategoryMember(userId = null) {
```

Comment:
This should not be called "isCategoryMember" - that implies it returns a boolean rather than doing things.

Reply:
okay i will make the changes